### PR TITLE
more 3.23 changes

### PR DESCRIPTION
**Pull request by lvlvllvlvllvlvl**

### CODE REVIEW

File: dat-schema/_Core.gql [hunk 1]
```diff
@@ -5710,6 +5831,31 @@ type ItemVisualReplacement {
   _: bool
 }
 
+# Added 3.23
+type LabyrinthCraftOptions {
```
Comment (SnosMe) [inline dat-schema/_Core.gql:5835]: already added in other file, please rebase

File: dat-schema/_Core.gql [hunk 2]
```diff
@@ -5348,6 +5385,47 @@ type AtlasMemoryLine {
...
+}
+
+# Added 3.23
+type azmerilifescalingperlevel {
+  _: i32
+  _: i32
+}
+
+# Added 3.23
+type azmeriwoodsdusttype {
```
Comment (SnosMe) [inline dat-schema/_Core.gql:5414]: This is league mechanic table, so should go into 3.23 file + should be properly CamelCased.

Comment (lvlvllvlvllvlvl) [reply dat-schema/_Core.gql]: done

File: dat-schema/_Core.gql (hunk 1 above)
Comment (lvlvllvlvllvlvl) [reply dat-schema/_Core.gql]: my bad

**Pull request merged**